pkg/planner/core/optimizer.go

```diff
@@ -504,6 +505,45 @@ func tryEnableLateMaterialization(sctx base.PlanContext, plan base.PhysicalPlan)
 	}
 }
 
+// tryEnableInvertedIndex tries to enable the inverted index for the table scan operator
+// FIXME: Now it is a naive implementation, we will refine it based on selectivity and cost in the future.
+func tryEnableInvertedIndex(sctx base.PlanContext, plan base.PhysicalPlan) base.PhysicalPlan {
```

Comment:
Now that the index is a MUST item. Don't do it in post optimize. Do it when generating the physical plan instead.

Reply:
OK, done